employees/tests/test_unit_export_data.py

```diff
@@ -87,6 +87,58 @@ def test_export_reports_for_project_should_download_for_project_csv_as_zip_file(
         self.assertEqual(response.status_code, 200)
         self.assertTrue(response._headers["content-disposition"][1].endswith('zip"'))
 
+    def test_export_reports_for_project_author_reports_should_download_if_user_is_manager_of_current_project(self):
+        manager = ManagerUserFactory()
+        self.project.managers.add(manager)
+        self.client.force_login(manager)
+        project_author_reports_url = reverse(
+            "export-project-author-reports-data-xlsx",
+            kwargs={
+                "pk": self.data["project"],
+                "user_pk": self.user.pk,
+                "year": str(self.report.creation_date.year),
+                "month": str(self.report.creation_date.month),
+            },
+        )
+
+        response = self.client.get(project_author_reports_url)
+        self.assertEqual(response.status_code, 200)
+
+    def test_export_reports_for_project_author_reports_should_download_csv(self):
+        manager = ManagerUserFactory()
+        self.project.managers.add(manager)
+        self.client.force_login(manager)
+        project_author_reports_url = reverse(
+            "export-project-author-reports-data-xlsx",
+            kwargs={
+                "pk": self.data["project"],
+                "user_pk": self.user.pk,
+                "year": str(self.report.creation_date.year),
+                "month": str(self.report.creation_date.month),
+            },
+        )
+
+        response = self.client.get(project_author_reports_url + "?format=csv")
+
+        self.assertEqual(response.status_code, 200)
+        self.assertTrue(response._headers["content-disposition"][1].endswith('csv"'))
+
+    def test_export_reports_for_project_author_reports_should_download_if_user_is_manager_of_current_project(self):
```

Comment:
this name is duplicated - probably it should have been: "if_user_is_**not**_manager"